tools/wptrunner/wptrunner/executors/executoratspi.py

```diff
@@ -11,13 +11,10 @@
 
 def poll_for_active_tab(root, logger, product):
     active_tab = find_active_tab(root, product)
-    iterations = 0
     while not active_tab:
         time.sleep(0.01)
         active_tab = find_active_tab(root, product)
-        iterations += 1
 
-    logger.debug(f"Found ATSPI active tab in {iterations} iterations")
```

Comment:
I actually thought we might as well leave this in here, it will only show up when debug logging.

Reply:
That's fair; I guess my thinking was that it's really only interesting if you're tuning the sleep time (which was why I added it), but I can imagine it might be useful if things start being really slow or something.